src/telemetry/telemetryController.ts

```diff
@@ -110,21 +110,21 @@ export default class TelemetryController {
     this._segmentAnalytics?.flush();
   }
 
+  public needTelemetry() {
+    return vscode.workspace.getConfiguration('mdb').get('sendTelemetry');
+  }
+
   public track(
     eventType: TelemetryEventTypes,
     properties: TelemetryEventProperties
   ): void {
-    const shouldSendTelemetry = vscode.workspace
-      .getConfiguration('mdb')
-      .get('sendTelemetry');
-
     log.info('TELEMETRY track', {
       eventType,
       segmentUserID: this._segmentUserID,
       properties
```

Comment:
I think when telemetry is disabled we don't want to show this log. How do you feel about moving it into the `if (this.needTelemetry()) {` below?